h2o-py/tests/testdir_jira/pyunit_pubdev_5174_isin_efficiency.py

```diff
@@ -0,0 +1,37 @@
+import sys
+sys.path.insert(1,"../../")
+import h2o
+from tests import pyunit_utils
+
+from h2o.exceptions import H2OResponseError
+from tests import pyunit_utils
+
+
+def pubdev_5174():
+
+    x = h2o.import_file(pyunit_utils.locate('smalldata/jira/PUBDEV-5174.csv'), header = 1)
+
+    tt = x['rr'].unique()
+    gg = tt[:10000, 0]
+
+    ww = x[~x['rr'].isin(gg['C1'].ascharacter().as_data_frame()['C1'].tolist())]
```

Comment:
ascharacter() probably has no effect as all the important values are already string ? Just noticed, probably not a big issue, if any.